doc/api/next_api_changes/behavior/22063-SR.rst

```diff
@@ -0,0 +1,7 @@
+Move Axes title to not overlap with y axis offset
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Axes titles that would overlap with y axis offset texts are now moved up if
+autopositioning is in effect (i.e. if *y* in `.Axes.set_title` is *None*
+and :rc:`axes.titley` is also *None*).
```

Comment:
```suggestion
autopositioning is in effect (i.e. if *y* in `.Axes.set_title` is *None*
and :rc:`axes.titley` is also *None*) and they overlap the offset text.
```

Reply:
Isn't this already said in the beginning of the main sentence: 

> Axes titles **that would overlap with y axis offset texts** are now moved up if autopositioning is in effect (i.e. if *y* in `.Axes.set_title` is *None* and :rc:`axes.titley` is also *None*).

I'm of course not an expert in English but for me the suggested addition sounds a bit redundant, doesn't it. The only thing I'm not sure is whether it should be "overlap with y axis offset text" or "overlap the y axis offset text".

Reply:
Ooops, you are right, that is redundant.  Maybe a slight rewording would be clearer:

```
Previously, Axes titles could overlap the y-axis offset text, which is often in the upper left corner of the axes.  Now titles are moved above the offset text if overlapping, and autopositioning is in effect (i.e. ...)
```

Reply:
Fixed. Thank you for you continuous support.